PSClassUtils/PSClassUtils.psm1

```diff
@@ -1,5 +1,18 @@
 $ScriptPath = Split-Path $MyInvocation.MyCommand.Path
 
+$PrivateClasses = gci "$ScriptPath\Classes\Private" -Filter *.ps1 | Sort-Object Name | Select -Expand FullName
+
+
+foreach ($Private in $PrivateClasses){
```

Comment:
Does this code really needs to be changed? It looks `just` like the one you erased.
You probably just added some line breaks, right?